core/gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -35,6 +35,7 @@ const disableInteractiveOption = {
   vertexAdd: false,
   vertexRemove: false,
   elementMove: false, // TODO: This is only a temporary change, will introduce another level of disable option.
+  addLinkFromMagnet: false,
```

Comment:
good call. @Xiao-zhen-Liu please verify this change.

Reply:
This is definitely needed. Good catch!